pkg/shell/plot.js

```diff
@@ -130,12 +130,16 @@ shell.plot = function plot(element, x_range_seconds, x_stop_seconds) {
     var options = { };
     var data = [ ];
     var flot = null;
-    var walk_timer;
     var result = { };
 
-    var now, x_range, x_offset, interval;
+    var interval;
+
+    var grid;
 
     function refresh() {
+        if (!grid)
```

Comment:
> Why would there ever be no grid? 

That was a consequence of making x_range optional in the constructor.  I made it mandatory again.